### PR TITLE
Added missing icons glyphs for open in terminal and hide protected SO files setting.

### LINKED ISSUE #5968: Open In Terminal context menu item missing an icon
### Description

The icon for the context menu item to open a folder in the terminal is missing an icon, and shows up as an empty square instead.

### Steps To Reproduce

_No response_

### Expected behavior

It should show a proper icon, instead of a blank square.

### Files Version

Version: 1.9.30.0

### Windows Version

Windows 11 22000.168 (Beta Channel)

### Relevant Assets

image

### LINKED ISSUE #5994: Files and folder settings has a missing icon on "hide system operative protected files" setting
### Description

There is no icon on the setting "hide system operative protected files" of Files and folder.

### Steps To Reproduce

1. Open settings.
2. Go to Files and Folders
3. Check that "hide system operative protected files" doesn't have an icon.

### Expected behavior

Should have an icon

### Files Version

Last main commit

### Windows Version

Windows 10 21H1 10.0.19043.1165

### Relevant Assets

image

### PR DESCRIPTION
**Pull request by BanCrash**
**Resolved / Related Issues**
- Closes #5968
- Closes #5994

**Details of Changes**
- Fixed the glyph icons for open in terminal and protected SO files setting.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
image
image

### CODE REVIEW

File: Files/Views/SettingsPages/FilesAndFolders.xaml [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
             <local:SettingsBlockControl Title="{helpers:ResourceString Name=SettingsFilesAndFoldersHideSystemItems}" HorizontalAlignment="Stretch">
                 <local:SettingsBlockControl.Icon>
-                    <FontIcon Glyph="&#xED1A;" />
+                    <FontIcon Glyph="&#xE72E;" />
```
Comment (yaichenbaum) [inline Files/Views/SettingsPages/FilesAndFolders.xaml:49]: I think we should use a path instead as the icon currently used is working on Windows 11.

Comment (gave92) [reply Files/Views/SettingsPages/FilesAndFolders.xaml:49]: Would conditional xaml work here? Would look different on w10 and 11 though.

Comment (BanCrash) [reply Files/Views/SettingsPages/FilesAndFolders.xaml:49]: > Would conditional xaml work here? Would look different on w10 and 11 though.

I had the same idea and currently I'm checking if we can use it!

Comment (BanCrash) [reply Files/Views/SettingsPages/FilesAndFolders.xaml]: Yep, it seems that there is not problem on using it!

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**